elements/order-input.html

```diff
@@ -67,9 +67,22 @@
                         if (header) header.hidden = true;
                     }
 
-                    var order = new parsers.OrderUpHtmlParser().parse(this.$.input);
-                    console.log('order', order);
-                    this._changeUrl(order);
+                    const parsersToTry = [parsers.OrderUpHtmlParser, parsers.OrderUpParser, parsers.CsvParser];
```

Comment:
Would `parsersToTry = Object.values(parsers)` work?

Reply:
That's what I originally had, but that would include the QueryStringParser, which only takes in a string, not an "elementOrText". (not a fan of the parser mutating the input, btw)

Reply:
I don't think QueryStringParser even uses its string argument. (I also don't see where input gets mutated).

Maybe we could add a flag and do this:

```javascript
var parsersToTry = Object.values(parsers).filter(p => !p.excludeFromInput)
```